assets/src/sass/base/_base.scss

```diff
@@ -1,17 +1,74 @@
-/*--------------------------------------------------------------
-# Base styles
---------------------------------------------------------------*/
-
-/* Reset
---------------------------------------------- */
-@import "resets";
-
-/* Typography
---------------------------------------------- */
-@import "typography/headings";
-@import "typography/typography";
-
-/* Elements
---------------------------------------------- */
-@import "elements";
-@import "forms";
+/*--------------------------------------------------------------
+# Base styles
+--------------------------------------------------------------*/
+
+/* Reset
+--------------------------------------------- */
+@import "resets";
+
+/* Typography
+--------------------------------------------- */
+@import "typography/headings";
+@import "typography/typography";
+
+/* Elements
+--------------------------------------------- */
+@import "elements";
+@import "forms";
+
+/* Layout (block style)
+--------------------------------------------- */
+:root {
+	--wp--style--global--content-size: 980px;
+	--wp--style--global--wide-size: 1020px;
+}
+.single-post {
+	--wp--style--global--content-size: 40rem;
+	--wp--style--global--wide-size: 980px;
```

Comment:
Oh, this is nice. I didn't realize changing content width could be done this easily. In another project we did it with a filter on "block_editor_settings_all".

Is there a risk in mixing units here? If a user changes the accessibility settings in their browser so that 1rem = 30px, the "content" width might be larger than the "wide" width.

Reply:
Interesting point. The content widths we are dealing with (posts content column and overall menu column width) _are_ designated in these exact units, so I think in this case things will behave the most consistently if we leave as-is. But I think we should refactor the overall site widths to either rems or px but not both when we hit the Refresh.